examples/gym/utils.py

```diff
@@ -32,7 +32,7 @@ def timestep_to_observations(timestep: dm_env.TimeStep) -> Mapping[str, Any]:
         for key, value in observation.items()
         if _WORLD_PREFIX not in key
     }
-  return gym_observations
+  return gym_observations, {}
```

Comment:
Why move this "info" to this method that reruns observations?
(also please update the type annotation)

Reply:
reset needs to also return info; I'll reverse this change and instead modify reset

Reply:
I'll also modify reset in the petingzoo example since I updated to gymnasium there as well but once again I did not test it